couchbase/example/src/main/java/example/springdata/couchbase/config/CouchbaseConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package example.springdata.couchbase.config;
+
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.couchbase.config.AbstractCouchbaseConfiguration;
+import org.springframework.data.couchbase.repository.config.EnableCouchbaseRepositories;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Couchbase Configuration to connect to Couchbase data store
+ *
+ * @author Chandana Kithalagama
+ */
+
+@SpringBootApplication
+@Configuration
+@EnableCouchbaseRepositories(basePackages = {"example.springdata.couchbase.repository"})
```

Comment:
Package declaration is not needed as it defaults to the base package the class resides in.